tests/e2e/flowtest/test_cephfilesystem_creation.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.helpers.helpers import create_resource, validate_cephfilesystem
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility import templating
+from ocs_ci.ocs.ocp import OCP
+
+
+logger = logging.getLogger(__name__)
+cephfs_data = templating.load_yaml(constants.CEPHFILESYSTEM_YAML)
+cephfs_data["metadata"]["name"] = "test-ceph-fs"
+cephfs_data["metadata"]["namespace"] = defaults.ROOK_CLUSTER_NAMESPACE
```

Comment:
Please use `constants.OPENSHIFT_STORAGE_NAMESPACE`, this is something from the past and we tend to not use it anymore.

Reply:
@shyRozen I have made the changes, could you please review it.